src/classes/CRLP_Rollup_TDTM.cls

```diff
@@ -172,13 +175,32 @@ public class CRLP_Rollup_TDTM extends TDTM_Runnable {
                 }
             }
 
-            // Account Rollup
-            if (!accountIds.isEmpty()) {
-                System.enqueueJob(new CRLP_RollupQueueable(accountIds));
-            }
-            // Contact Rollup
-            if (!contactIds.isEmpty()) {
-                System.enqueueJob(new CRLP_RollupQueueable(contactIds));
+            // Execute the roll-ups in real-time instead of asynchronously in the following cases:
+            // - Currently in a Unit Test with only a couple of summary objects (unit tests do not support chaining)
+            // - Called from a Batch Job, @Future or Queueable context
+            //  || System.isBatch() || System.isFuture() || System.isQueueable()
```

Comment:
Why were the the extra batch/future/queueable conditions removed?

Reply:
I had removed them for some testing and forgot to add them back in. Interesting that having them not in there didn't create any issues in my test rollups. Maybe I just didn't hit a scenario where these conditions were met. I put them back in.